src/main/java/com/airlines/model/Crew.java

```diff
@@ -0,0 +1,46 @@
+package com.airlines.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Crew {
+    private int id;
```

Comment:
fields could be final

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/ca953f2c0ce7056d2c0a27ab1ec94814c001ae75